File: examples/access/constraints/look_direction_left.py
# /// script
# dependencies = ["brahe"]
# ///
"""
Create a look direction constraint requiring left-looking geometry.
"""

import brahe as bh
from brahe import LookDirection

# Require left-looking geometry
constraint = bh.LookDirectionConstraint(allowed=LookDirection.LEFT)

print(f"Created: {constraint}")
# Created: LookDirectionConstraint(Left)

File: examples/access/constraints/look_direction_left.rs
//! Create a look direction constraint requiring left-looking geometry.

#[allow(unused_imports)]
use brahe as bh;

fn main() {
    // Require left-looking geometry
    let constraint = bh::LookDirectionConstraint::new(
        bh::LookDirection::Left
    );

    println!("Created: {}", constraint);
    // Created: LookDirectionConstraint(Left)
}
